dnf-behave-tests/fixtures/specs/needs-restarting/test-1.0-1.fc29.spec

```diff
@@ -0,0 +1,21 @@
+Name:           test
+Version:        1.0.0
+Release:        1
```

Comment:
Nitpick: the release actually doesn't contain "fc29", which you have in the filename, you could remove it, but whatever... :slightly_smiling_face: